spec/integration/profiles_spec.rb

```diff
@@ -10,7 +10,7 @@
     policies(:one).update! account: accounts(:one)
   end
 
-  path "#{Settings.path_prefix}/#{Settings.app_name}/profiles" do
+  path '/profiles' do
```

Comment:
It seems to request from root, ignoring the base path, for me.
```
http://localhost:3000/profiles?...
https://ci.foo.redhat.com:1337/profiles?...
```
Maybe the opening `/` slash should be omitted to get the right base?

```
Using rswag-api 2.3.3
Using rswag-specs 2.3.3
Using rswag-ui 2.3.3
```

Reply:
I'm not sure why it's not working for you. If you check other openapi.json from other apps, paths are specified with leading slashes. Remediations, for example:

```js
{
  "openapi": "3.0.0",
  "info": {…},
  "servers": […],
  "tags": […],
  "paths": {   
    "/remediations": {…},
    "/remediations/download": {…},
    "/remediations/{id}": {…},
    ...
    "/playbook": {…},
    "/version": {…}
  },
  "components": {…}
}
```

I think basePath may not be the correct way to specify this, though. I saw it in approval's openapi.json, but it looks like most other services specify the base path in the `servers`. I will update this PR with that to see if it fixes your issue.

Reply:
See http://spec.openapis.org/oas/v3.0.3#paths-object